security/rs-security-best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Best Practices
+page_title: Best Security Practices when Installing and Using Telerik Report Server
+description: "Check the recommended security practices when installing the Telerik Report Server and working with its reports, users, and services."
+slug: rs-security-best-practices
+tags: telerik, reporting, report, server, security, best, practices
+published: True
+position: 4
+---
+
+# Security Best Practices
+
+The article discusses general and report server-specific security practices. While the recommendations may be necessary to keep your Telerik Report Server instance with all report definitions, data connections, etc. secure, they may not be sufficient. The article should not be regarded as a complete and comprehensive security guidance.
```

Comment:
The introduction is a bit vague. We say "... may not be sufficient", but there's no guidance what to do in this case.  Also, the purpose of the article is exactly to be regarded as a security guidance, as comprehensive as we can do in the context of RS. For the rest they should consult a specialist.

How about this one:
This article covers security practices that are generally applicable to web applications, as well as those specific to Telerik Report Server. The recommendations below should be regarded as a security guidance that ensures your information stored in Telerik Report Server is protected and safe. We recommend consulting with a specialist to provide further assistance on web application safety.